### ISSUE TITLE
Error with keyvault after upgrade to 0.16.8.

### ISSUE THREAD

**Issue by oliverholliday** [pulumi/pulumi-azure#180]
I've just updated to 0.16.8 from 0.16.7 and now i'm getting:

```
~   ├─ azure:keyvault:AccessPolicy     pulumi-keyvault-create-access    refresh  [diff: +keyVaultId]
```

followed by

```
azure:keyvault:Secret (EventHubLeaseStorage):
    error: Preview failed: refreshing urn:pulumi:operations::operations::azure:keyvault/secret:Secret::EventHubLeaseStorage: 
Error checking if key vault "" for Secret "EventHubLeaseStorage" in Vault at url "https://pulumi.vault.azure.net/" exists: keyVaultId is empty
```

I've added this to the `azure.keyvault.Secret` definition but no joy.

```
keyVaultId: mykeyvault.id
```

**Comment by lukehoban** [comment]
This is being tracked in https://github.com/terraform-providers/terraform-provider-azurerm/pull/2820.

**Comment by jen20** [comment]
This is fixed by v0.16.9 of the provider - the packages are being released at the moment, and should be out shortly. Thanks for the report, @oliverholliday!
